packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -634,6 +634,26 @@ class TreemapNode extends TreeNode<TreemapNode> {
     return path.reversed.toList();
   }
 
+  /// Returns the path for this node, including only package nodes (nodes that
```

Comment:
nit: text up to the comma could  be the first line in the comma with the text after the comma as a description providing more information.

Reply:
done